src/pages/ScreeningsPage.vue

```diff
@@ -57,7 +70,25 @@ export default {
       return `${dayName}, ${date}`;
     },
     moviesOnTheDay() {
-      return [...new Set(this.seancesOnTheDay.map((seance) => seance.movie))];
+      const moviesShown = [
+        ...new Set(this.seancesOnTheDay.map((seance) => seance.movie)),
+      ];
+      const moviesWithGenre = moviesShown.map((movie) => {
+        return {
+          id: movie,
+          genre: this.movieList.find((film) => film.id === movie).genre.name,
+        };
+      });
+      return moviesWithGenre;
+    },
+    moviesByCategory() {
+      if (this.selectedGenre === "") {
+        return this.moviesOnTheDay;
+      } else {
+        return this.moviesOnTheDay.filter(
+          (movie) => movie.genre === this.selectedGenre
+        );
+      }
```

Comment:
A small thing, but if you have a simple return in the `if () { }` condition, and a single `else` case then no need to wrap things with the `else { }` 💡 
```suggestion
      }
      return this.moviesOnTheDay.filter(
        (movie) => movie.genre === this.selectedGenre
      );
```

Reply:
thiis is very true 😨 thank you!!